src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
Ah, I've missed that there are two kinds of encoding - one is direct size in the UnwindData  and the other is reading it from memory - the *(PTR_ULONG)(ImageBase + FunctionLength) & 0x3ffff.